sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -626,20 +623,21 @@ public async Task UpdateAzureApplicationInsightsDataFeedWithEveryMemberAndGetIns
             MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
 
             var dataFeedName = Recording.GenerateAlphaNumericId("dataFeed");
+            var updatedDataFeedName = Recording.GenerateAlphaNumericId("dataFeed");
```

Comment:
Since we can freely set the data feed name now (not restricted by the constructor), we are adding validation for updating the name as well.